db/schema_applier.hh

```diff
@@ -44,6 +44,13 @@ struct schema_complete_view {
     schema_tables::schema_result scylla_aggregates;
 };
 
+// groups keyspaces based on what is happening to them during schema change
+struct affected_keyspaces {
```

Comment:
it's not in global namespace, it's in db::schema_tables, I agree that it could be in schema_applier, but that makes little sense on it's own. In fact most functions called by `schema_applier::update` should be just class methods, but I prefer not to do it for now, it's a lot of changes not contributing that much so it'd justify making this PR bigger.